controller/gif-controller.js

```diff
@@ -0,0 +1,95 @@
+const db = require("../db/db");
+const cloudinary = require("cloudinary");
+require("dotenv").config();
+
+const createGif = async (req, res) => {
+  // get the token details from the authmiddleware
+  const user = req.user;
+  const { base64EncodedGif, title } = req.body;
+
+  // configure cloudinary
+  cloudinary.config({
+    cloud_name: process.env.CLOUD_NAME,
+    api_key: process.env.CLOUD_API_KEY,
+    api_secret: process.env.CLOUD_API_KEY_SECRET,
+  });
```

Comment:
Do This Configuration In Another File , Then Import Here

Reply:
done